exporter/exporter.go

```diff
@@ -370,6 +385,14 @@ func NewRedisExporter(uri string, opts Options) (*Exporter, error) {
 			"search_total_indexing_time": "search_indexing_time_ms_total",
 			"search_total_cycles":        "search_cycles_total",
 			"search_total_ms_run":        "search_run_ms_total",
+			// RediSearch module v8.0
+			"search_gc_total_cycles":               "search_gc_cycles_total", // search_gc metrics were renamed into user and interna
+			"search_gc_total_ms_run":               "search_gc_run_ms_total", // in PR: https://github.com/RediSearch/RediSearch/pull/5616
```

Comment:
We can name these metrics same way as they were in pre-v8.0, like `search_cycles_total` instead of `search_gc_cycles_total`, but I would say new naming clearly states that these metrics are related to garbage collection (gc) and there is one more new metric: `search_gc_docs_not_collected_total`, which won't match old pattern.

Reply:
+1 on `search_gc_*`